src/test/java/com/google/cloud/tools/appengine/operations/AppYamlProjectStagingTest.java

```diff
@@ -357,4 +358,54 @@ public void testFindRuntime_malformedAppYaml() throws IOException {
       assertEquals("Malformed 'app.yaml'.", ex.getMessage());
     }
   }
+
+  private static final Path complexLib = Paths.get("src/test/resources/jars/complexLib.jar");
+  private static final Path simpleLib = Paths.get("src/test/resources/jars/libs/simpleLib.jar");
+  private static final Path complexLibBadManifest =
+      Paths.get("src/test/resources/jars/complexLibBadManifest.jar");
+  private static final Path missingLib = Paths.get("src/test/resources/jars/libs/missing.jar");
+
+  @Test
+  public void testCopyArtifactJarClasspath_noClasspath() throws IOException, AppEngineException {
+    artifact = simpleLib;
```

Comment:
This is strange. I don't think you need the artifact field or the static final Path fields here. You can move each path into the single method that uses that one path.

Reply:
yeah that was a refactor gone wrong. good catch. the static finals are still used in verification, but I definitely don't need artifact or config to use the pre-initialized values.

Reply:
I guess simpleLib is the only one that is repeated everywhere, I could potentially get rid of the static finals